### ISSUE TITLE
Cannot include remote.min.js

### ISSUE THREAD

**Issue by pljspahn** [guillaumepotier/Parsley.js#859]
Uncaught TypeError: undefined is not a function -- parsley.remote.min.js:1 (anonymous function)

Guessing that you are not properly using $ symbol. The page I am including parsley on also has prototype and jQuery loaded, setting `$ = jQuery` is not an appropriate fix.

**Comment by marcandre** [comment]
`parsley` relies on `jQuery`, not `$`. Fiddle example [here](http://jsfiddle.net/0arpdps3/) works fine, so it should be something else.
Make sure you load jQuery beforehand, and try the non compressed version to get a meaningful line number.

**Comment by marcandre** [comment]
Oh, you're right, `remote` has that problem, I had tried with the non-remote version.

**Comment by marcandre** [comment]
Fixed in v2.1.0-rc4
Thanks for reporting this!

**Comment by pljspahn** [comment]
Yeah, I guess I must not have a clue what I'm doing. I cannot get any bits of this plugin to work now. 

Include jQuery, include parsley.remote, include parsley, call `jQuery('element').parsley()` and it's "undefined is not a function" no matter how I set it up.

I'd love to be able to use this plugin, but at this point I've spent too many hours trying to do the equivalent of "hello world". It's a shame, because it seems like a nice plugin. Oh well.
